app/src/main/res/layout/layout_action_buttons.xml

```diff
@@ -9,10 +9,12 @@
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/recyclerViewActionButtons"
-        android:layout_width="match_parent"
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_gravity="center"
         android:orientation="horizontal"
-        app:layoutManager="com.google.android.flexbox.FlexboxLayoutManager"
+        android:overScrollMode="never"
```

Comment:
🤯 how is this possible??

Reply:
what is you mean? short list, overScroll not need

Reply:
i meant that you replaced FlexboxLayoutManager with LinearLayoutManager and now it works okay? can you send the screenshot of how the buttons look like now?

Reply:
![photo_2023-01-06 12 00 39](https://user-images.githubusercontent.com/121934748/210956796-0dc74a96-9f5c-472a-b746-b20f4f58435a.jpeg)
![photo_2023-01-06 12 00 33](https://user-images.githubusercontent.com/121934748/210956803-d910ff26-03bd-4498-81dd-953335f1eb9b.jpeg)
![photo_2023-01-06 12 00 30](https://user-images.githubusercontent.com/121934748/210956807-b113a5c8-e327-4b05-aec9-e7b4e00abab8.jpeg)